config/settings.yml

```diff
@@ -223,15 +222,15 @@ modules_appeals_api:
     appeals_api_hlr_cleanup_pii:
       class: AppealsApi::HigherLevelReviewCleanUpWeekOldPii
       description: "Remove PII of HigherLevelReviews"
-      every: ['24h', first_in: '45m']
+      every: ["24h", first_in: "45m"]
     appeals_api_hlr_status_batch:
       class: AppealsApi::HigherLevelReviewUploadStatusBatch
       description: "Update HigherLevelReview statuses with their Central Mail status"
       every: "60m"
     appeals_api_nod_cleanup_pii:
       class: AppealsApi::NoticeOfDisagreementCleanUpWeekOldPii
       description: "Remove PII of NoticeOfDisagreements"
-      every: ['24h', first_in: '45m']
+      every: ["24h", first_in: "45m"]
```

Comment:
These quotes and formatting changes don't seem related to the PR. Could you remove these please?